### DISCUSSION TITLE
Welcome to Research Discussions!

### DISCUSSION THREAD

**Discussion by AnatoNamikaza** [AnatoNamikaza/Research#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zaeemyousaf** [comment]
what is your field of interests?

**Reply by AnatoNamikaza** [reply]
Anything related to this cutting edge tech

**Comment by zaeemyousaf** [comment]
I once saw your profile relating to research in Quantum Computing. Are you still active?

**Reply by AnatoNamikaza** [reply]
Yes it is, I am currently researching on quantum and its applications in bio-informatics

**Comment by AnatoNamikaza** [comment]
Which is the best language in the world right now?

**Reply by zaeemyousaf** [reply]
If you are an evil than LISP, if you are an angel than Python else English.
